src/editor/utils.js

```diff
@@ -49,3 +54,44 @@ export const getEditPostPayload = ( {
  * @return {boolean} True if it contains a valid email string.
  */
 export const hasValidEmail = string => /\S+@\S+/.test( string );
+
+/**
+ * Get interest settings for a Mailchimp campaign.
+ * An interest is a subset of a list.
+ */
+export const getListInterestsSettings = ( { campaign, interestCategories } ) => {
+	if (
+		! interestCategories ||
+		! interestCategories.categories ||
+		! interestCategories.categories.length
+	) {
+		return;
+	}
+	const options = interestCategories.categories.reduce( ( accumulator, item ) => {
+		const { title, interests, id } = item;
+		accumulator.push( {
+			label: title,
+			disabled: true,
+		} );
+		if ( interests && interests.interests && interests.interests.length ) {
+			interests.interests.forEach( interest => {
+				const isDisabled = parseInt( interest.subscriber_count ) === 0;
+				accumulator.push( {
+					label:
+						'- ' +
+						interest.name +
```

Comment:
While we're at it should the Interests/Groups `SelectControl` display subscriber count?

Reply:
Here you go: 5dc7c00a807466e1c195ede3898f6fc61ad1ea8e